act-rules-format.bs

```diff
@@ -245,183 +263,210 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
 Applicability MUST be described objectively, unambiguously and in plain language.
 
-An objective description is one that can be resolved without uncertainty, in a given technology. Examples of objective properties in HTML are element names, their computed role, the spacing between two elements, etc. Subjective properties on the other hand, are concepts like decorative, navigation mechanism and pre-recorded. Even concepts like headings and images can be misunderstood. For example, describing that the rule examines the tag name, the accessibility role, or the element's purpose on the web page. The latter of which is almost impossible to define objectively. When used in applicability, potentially ambiguous concepts MUST be defined objectively. This definition can be part of a larger glossary shared between rules.
+An objective description is one that can be resolved without uncertainty, in a given technology. Examples of objective properties in HTML are tag names, their computed role, the distance between two elements, etc. Subjective properties on the other hand, are concepts like decorative, navigation mechanism and pre-recorded. Even concepts like headings and images can be misunderstood. These terms could refer to the tag name, the semantic role, or the element's purpose on the web page. The latter of which is almost impossible to define objectively. When used in applicability, potentially ambiguous concepts MUST be defined objectively. Definitions can be put in the rule [glossary](#glossary), or they can be defined in the section they are used.
 
 <div class=example>
-  <p><strong>Example</strong>: The applicability of an atomic rule testing <a href="https://www.w3.org/WAI/WCAG21/quickref/#audio-control" target="_blank">WCAG 2.1 Audio Control:</a></p>
+  <p>**Example:** The applicability of an atomic rule testing [WCAG 2.1 Audio Control](https://www.w3.org/WAI/WCAG21/quickref/#audio-control):</a></p>
   <blockquote>
-    <p>Any <code>video</code> or <code>audio</code> element(s) with the <code>autoplay</code> attribute, as well as any <code>object</code> element(s) that is used for automatically playing video or audio when the web page loads.</p>
+    <p>Each `video` or `audio` element with the `autoplay` attribute, as well as each `object` element that is used for automatically playing video or audio when the web page loads.</p>
+    <p>**Note: A web page is considered "loaded" when the `document.readyState` is set to `complete`.</p>
   </blockquote>
 </div>
 
 
-Applicability for Composite Rules {#applicability-composite}
-----------------------------------------------------------
+### Applicability for Composite Rules ### {#applicability-composite}
 
-A [=composite rule=] defines how the [=outcomes=] from rules in its [input rules list](#input-rules-list) are used to determine a single outcome.
+A [=composite rule=] defines how the [=outcomes=] from atomic rules in its [input rules](#input-rules) are used to determine a single [=outcome=].
```

Comment:
**Editorial**: I'm not sure this intro paragraph adds anything new. We've explained this several times in the spec, and in applicability, this isn't relevant I think. I think removing this improves readability.